gramps/gui/views/treemodels/familymodel.py

```diff
@@ -116,63 +117,63 @@ def on_get_n_columns(self):
         return len(self.fmap) + 1
 
     def column_father(self, data):
-        handle = data[0]
+        handle = data["handle"]
```

Comment:
add constants and use instead of raw strings throughout file?

Reply:
See note.